api/src/main/java/openconsensus/trace/SpanContext.java

```diff
@@ -41,7 +41,7 @@
    *
    * @since 0.1.0
    */
-  public static final SpanContext INVALID =
+  static final SpanContext INVALID =
```

Comment:
I think it should move to SDK package.

Reply:
So we need something here for `BlankSpan.getContext()` - so we could put it there and rename it (`BLANK_CONTEXT` perhaps?).

Also, was forgetting that `TraceId` and `SpanId` expose an `INVALID` instance each - sounds like we should hide/remove them as well, @bogdandrutu ?

Reply:
Based on w3c https://github.com/w3c/trace-context/blob/master/spec/20-http_header_format.md#trace-id these value are considered invalid, so I think it is fine to export them. w3c does not mention about invalid SpanContext so happy to remove that.

Reply:
Makes sense. I will then remove `SpanContext.INVALID` and put an internal instance in `BlankSpan`, while not touching Span/Trace ids.